src/models/userModel.ts

```diff
@@ -0,0 +1,22 @@
+import mongoose from "mongoose";
+const userSchema = new mongoose.Schema({
+  name: {
+    type: String,
+    required: true,
+  },
+  email: {
+    type: String,
+    required: true,
+    unique: true,
+  },
+  password: {
+    type: String,
+    required: true,
```

Comment:
password should have the select prop set to false to only send the password if required.